input/system/rds/system.go

```diff
@@ -36,11 +36,6 @@ func GetSystemState(config config.ServerConfig, logger *util.Logger) (system sta
 		return
 	}
 
-	if instance == nil {
-		logger.PrintWarning("Could not find RDS instance in AWS, skipping system data")
-		return
-	}
```

Comment:
This code is no longer needed since we now always return an error when we can't find the instance, instead of only returning `instance = nil`.